csv-data-management/extract_data_from_csv.py

```diff
@@ -200,6 +200,119 @@ def merge_two_csv_and_extract_data(config, params):
         logger.error('Error in extract_data_from_csv(): %s' % Err)
 
 
+def concat_two_csv_and_extract_data(config, params):
+    try:
+        numberOfRowsToSkipFirst = None
+        numberOfRowsToSkipSecond = None
+        isSingleColumnFrist = None
+        isSingleColumnSecond = None
+        isCSVWithoutHeadersFirst = False
+        isCSVWithoutHeadersSecond = False
+
+        file_1_iri_ = handle_params(params,params.get('file_one_value'))
```

Comment:
file_1_iri --> fileOneIri
file_2_iri --> fileTwoIri

Reply:
I will stay with current nomeclature if is not required , from readability perspective